monitor/src/main/webapp/WEB-INF/views/admin/addApp.jsp

```diff
@@ -148,12 +222,78 @@
 		}
 	}
 	
+	function populateTags() {
+		var tags = "${appForm.tagString}".split(",");
+		for (var i = 0; i < tags.length; i++) {
```

Comment:
I think I would iterate over the tags in the template and then just call `updateTagsField` for each instead of doing this with javascript. Using comma to separate tags, makes it wonky (what if someone uses a comma in a tag).